VelocityCore/src/main/java/test/ConversationTest.java

```diff
@@ -0,0 +1,17 @@
+package test;
+
+import de.bentzin.conversationlib.manager.ConversationManager;
+import net.kyori.adventure.audience.Audience;
+
+/**
+ * @author Ture Bentzin
+ * 25.03.2023
+ */
+public class ConversationTest {
+    public static void demo() {
```

Comment:
![grafik](https://static.wikia.nocookie.net/minecraft_gamepedia/images/7/72/Demoshot2.png/revision/latest/scale-to-width-down/372?cb=20170510072034)

Reply:
removed with bff09bdf01ad43c9d9bf6f0db11d7cbbf1c0979e